config/etc/system-image/writable-paths

```diff
@@ -42,35 +33,29 @@
 # needed by apparmor - use transition since some core apps are
 # pre-installed on the image
 /var/cache/apparmor                     auto                    persistent  transition  none
+# FIXME: do we still need this?
 /var/lib/apparmor                       auto                    persistent  transition  none
-/var/lib/whoopsie                       auto                    persistent  transition  none
 # required by update-initramfs
 /var/tmp                                auto                    persistent  transition  none
 # ssh
 /etc/ssh                                auto                    persistent  transition  none
-/etc/init                               auto                    persistent  transition  none
 # used for various writable files (timezone, localtime, ...)
 /etc/writable                           auto                    synced      none        none
-# ureadahead
-/var/lib/ureadahead                     auto                    persistent  transition  none
+# FIXME: why do we need this?
 # required by update-initramfs
 /var/lib/initramfs-tools                auto                    persistent  transition  none
 # systemd
 /etc/systemd/system                     auto                    persistent  transition  none
-# needed for usb tethering
-/var/lib/misc                           auto                    persistent  transition  none
-# walinuxagent
-/var/lib/waagent                        auto                    persistent  transition  none
 # ssh keys
 /root                                   auto                    persistent  transition  none
 # passwordless sudo
 /etc/sudoers.d                          auto                    persistent  transition  none
 /etc/hosts                              auto                    persistent  transition  none
 /var/lib/extrausers                     auto                    persistent  transition  none
+# FIXME: do we still need this?
```

Comment:
Would be great to drop this but we may have users who have files in there after they migrated from a 15.04 installation. Not sure what an update would do or if those people are still using them.

Reply:
I think it would need to stay, snap create-user and other adduser/useradd writes there when adding users (system or otherwise).

Reply:
I'm pretty sure the FIXME refers to the line below, not above ;)

Give that we only ship netplan (or netwok-manager via the model assertion) and ifupdown is completely gone from the images, i guess that dir can go as well...